Description: re-enable AutoUpdate at waagent.conf
 After having disabled AutoUpdates it was agreed with Azure that
 the ubuntu package was required to re-enable them by default.
Author: Jose Vazquez <jose.vazquez@canonical.com>
Origin: upstream
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1650522
Forwarded: not-needed
Last-Update: 2016-12-16

Index: walinuxagent-2.1.5/config/ubuntu/waagent.conf
===================================================================
--- walinuxagent-2.1.5.orig/config/ubuntu/waagent.conf
+++ walinuxagent-2.1.5/config/ubuntu/waagent.conf
@@ -82,5 +82,5 @@ OS.OpensslPath=None
 # OS.EnableRDMA=y
 
 # Enable or disable self-update, default is enabled
-AutoUpdate.Enabled=n
+AutoUpdate.Enabled=y
 AutoUpdate.GAFamily=Prod
